### DISCUSSION TITLE
Countries inside of Britain should have separate flags

### DISCUSSION THREAD

**Discussion by rennk** [ppy/osu#14500] [Ideas]
This is far fetched but please hear me out. The UK population consists of about 70 million people. 60 million of which are from England (86%), 5 million from Scotland (7%) , 2 million from Northern Ireland (2.8%) and 3 million from Wales (4.3%). I believe that having a way to distinguish what part of the UK you are from would allow for the players from these places to find a better sense of community. It would also be interesting to have a separate leaderboard for each country inside of the UK. Places like Palau have their own leaderboard although only 27 players from Palau exist and their population is only around 20,000. If this change was implemented then i think keeping the whole of the UK competing in OWC and such, similar how the Olympics handle this issue would be the correct approach. This would be a great service to those who are not from England as many of us feel not only unrepresented, but also misrepresented by the British flag. Thanks a lot.

**Comment by LittleEndu** [comment]
Allowing this change sounds like an easy way for people to start requesting similar changes like "All US states should have separate flags" in an never ending loop. 

Unless you suggest a list of territories from an authority (like you mention Olympics) that happens to have UK territories as separate, just saying "separate please" isn't going to be enough. Pretty sure the current list is taken from some ISO or web standard anyway and is very unlikely to be changed.

**Reply by bdach** [reply]
The UK is a bit of a special case. There is precedent for this in other sports (i.e. association football which doesn't have a "Great Britain" national team outside of the Olympics).

But this should go under discussion in osu-web first, probably.

**Reply by rennk** [reply]
I also agree that the UK is a special case, it isn't even comparable to the likes of states in the US. I'm unaware of any other situations like the UK. I dont know what osu-web is apologies  if this is the wrong place to post.

**Reply by gagahpangeran** [reply]
Open this issue at https://github.com/ppy/osu-web/

**Comment by gagahpangeran** [comment]
I think this will affect OWC (and another country based tournament), because the members of the team should be from the same 'country' or 'flag'.

But dont take my words because I'm not OWC staff.

**Reply by rennk** [reply]
I think a system could be put in place pretty easily where the UK compete together for OWC, but the countries inside the UK can also have their separate leaderboards.

**Reply by honguyenminh** [reply]
I think that's a bit over-engineered, it requires special case handling just for a country, and add complexity to leaderboard system

**Reply by rennk** [reply]
For a population of 10 million people i dont think this should be just glanced over. it could do good things for the osu! scene in the uk's countries
